assets/helpers/abTests/abtest.js

```diff
@@ -123,6 +123,27 @@ function getParticipationsFromUrl(): ?Participations {
   return null;
 }
 
+function getParticipationsFromQuery(): ?Participations {
+
+  const hashUrl = (new URL(document.URL)).search;
+  const index = hashUrl.indexOf('ssr');
```

Comment:
not sure if I should be more careful in case some of the other tracking stuff happens to have 'ssr' in the string.